velero-plugin-for-gcp/object_store.go

```diff
@@ -76,7 +77,7 @@ func newObjectStore(logger logrus.FieldLogger) *ObjectStore {
 }
 
 func (o *ObjectStore) Init(config map[string]string) error {
-	if err := veleroplugin.ValidateObjectStoreConfigKeys(config, kmsKeyNameConfigKey, serviceAccountConfig); err != nil {
+	if err := veleroplugin.ValidateObjectStoreConfigKeys(config, kmsKeyNameConfigKey, serviceAccountConfig, caCert); err != nil {
```

Comment:
the plugin framework function has been updated to always allow this key - see https://github.com/vmware-tanzu/velero/blob/master/pkg/plugin/framework/validation.go#L30. So we shouldn't have to add the key here once we can bump the version of the dependency.

I do think we should probably change https://github.com/vmware-tanzu/velero/blob/master/pkg/persistence/object_store.go#L115-L117 to only add these items to the `Config` map if they're non-empty since we'll likely hit this issue for community plugins when we release the ca cert change.

Reply:
Got it, thanks. I saw this when I was testing against a branch based off `master`, so the GCP plugin hadn't been updated yet. I didn't catch the plugin framework part of the change, but I agree that's a better place to put it than updating each plugin individually.

I'll close out this PR.